web/src/app/pages/create-survey/create-survey.component.spec.ts

```diff
@@ -40,22 +44,52 @@ describe('CreateSurveyComponent', () => {
   let route: ActivatedRouteStub;
   let activeSurvey$: Subject<Survey>;
   let surveyServiceSpy: jasmine.SpyObj<SurveyService>;
+  let jobServiceSpy: jasmine.SpyObj<JobService>;
 
   const surveyId = 'survey001';
   const newSurveyId = 'survey002';
   const title = 'title1';
   const description = 'description1';
+  const jobId = 'job001';
+  const name = 'jobX';
   const mockSurvey = new Survey(
     surveyId,
     title,
     description,
     /* jobs= */ Map(),
-    /* acl= */ Map()
+    /* acl= */ Map(),
+    OnboardingStatus.NAME_SURVEY
+  );
+  const mockSurveyNoJob = new Survey(
```

Comment:
Nit: These aren't mocks, they're real objects, so you can remove the "mock" prefix in var names.

Reply:
Done.